docs/api/web-contents.md

```diff
@@ -1160,26 +1160,26 @@ one through the `'paint'` event.
 
 #### `contents.id`
 
-The unique ID of this WebContents.
+A Integer representing the unique ID of this WebContents.
```

Comment:
Returns an Integer representing the unique ID of this WebContents instance.

Should generate: `readonly id: number`.